lstchain/_version_cache.py

```diff
@@ -0,0 +1 @@
+version='0.7.5.post2+git36a8068'
```

Comment:
I think this file should not have been committed.

Reply:
yes, fixed. thanks for spotting